recipes/fast-dds/all/conanfile.py

```diff
@@ -146,7 +146,6 @@ def validate(self):
                 # linking dynamic '*.dll' and static MT runtime
                 raise ConanInvalidConfiguration("Mixing a dll {} library with a static runtime is a bad idea".format(self.name))
 
-
```

Comment:
This modification in conanfile is unecessary. Remove it and CI could detect a bump.

Reply:
I dont think the bot can since there is a patch... Unless it got wicked smart (there's some smart people on the Conan team)

Reply:
it can, thanks to:

```python
    def export_sources(self):
        self.copy("CMakeLists.txt")
        for patch in self.conan_data.get("patches", {}).get(self.version, []):
            self.copy(patch["patch_file"])
```

Reply:
🤯 I'll need to improve my bot then! Thanks for the news